avro2tf/src/main/scala/com/linkedin/avro2tf/jobs/TensorMetadataGeneration.scala

```diff
@@ -34,13 +33,23 @@ class TensorMetadataGeneration {
       getColsOfIntOrLongCardinalityMapping(dataFrame, params) ++
       getColsWithHashInfoCardinalityMapping(params)
 
-    val featuresTensorMetadata = generateTensorMetadata(params.tensorizeInConfig.features, colsToFeatureCardinalityMapping)
-    val labelsTensorMetadata = generateTensorMetadata(params.tensorizeInConfig.labels.getOrElse(Seq.empty), colsToFeatureCardinalityMapping)
+    val featuresTensorMetadata = generateTensorMetadata(
```

Comment:
qq. This is auto-formatting? (also below)

It's a little hard to review because it is not super clear for the real changes. Anything we can do about this? :)

Reply:
just double checked, it's caused by auto formatting and it is our photon style. Sorry for causing the difficulties to review this PR